make_pyne_docker_image.sh

```diff
@@ -0,0 +1,112 @@
+#! /bin/bash
+
+#_____________________________________________________________________________
+# usage
+# call if option -h, -help or --help, display usage and quit
+function usage ()
+{
+echo "---------------------------------------------------------------------------"
+echo "---------------------- make_pyne_docker_image usage: ----------------------"
+echo " "
+echo "  -h|-help|--help: print help/usage "
+echo " "
+echo "  --moab|-moab|moab : add MOAB to the build "
+echo " "
+echo "  --pymoab|-pymoab|pymoab : add pyMOAB to the build (also enable MOAB) "
+echo " "
+echo "  --dagmc|-dagmc|dagmc : add DAGMC to the build (also enable MOAB) "
+echo " "
+echo "  --deps-only|-deps-only|deps-only|--deps|-deps|deps : only build "\
+    "dependencies (PyNE will not be build/installed)"
+echo " "
+echo "  -tag=*|--tag=*|tag=*|-t=*|--t=*|t=*: set the tag name of the "\
+            "created docker image, if not provided tag name will be form from "\
+            "the install option : 
+            \"ubuntu_18.04(_pyne/_pyne-deps)(moab/pymoab/)(dagmc/) \""
+echo " "
+echo "---------------------------------------------------------------------------"
+exit 418 
+}
+
+
+# Parsing arguments
+MOAB=false
+DAGMC=false
+PYMOAB=false
```

Comment:
I think we should just default to build all of these (set to True). Any reason why the default is false?

Reply:
my though was to be additive with the option rather than removing option...

Reply:
Rather than thinking of it as additive vs removing, we should simply think of how we think we'll use it most often - probably with all of these enabled.

That said, the syntax of command-line options that remove things is a little more cumbersome: `--no-dagmc`?